stlab/channel.hpp

```diff
@@ -570,6 +593,11 @@ class sender {
         auto p = _p.lock();
         if (p) p->send(std::forward<A>(args)...);
     }
+
```

Comment:
Senders cannot be throttled so the sender queue size is always unlimited. We want the queue size to go between processes. Something like:

```
   auto receiver = r | buffer_size(16) | [](int x) { ... };
```
The idea is similar to stream io manipulators. I would use "buffer_size" instead of queue_size to match the terminology from the Go language.